libs/wingsdk/src/std/util.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Generic type argument. This type is replaced at compile time.
```

Comment:
Should this be `@hidden` somehow so it cannot be instantiated?

Reply:
Yes is should, I didn't put it at first because for some reason I thought it would break compilation. Will add.